integrations/extensions/docs/elasticsearch-install-and-setup/python-document-ingestion/README.md

```diff
@@ -0,0 +1,207 @@
+pip3 install -r requirements.txt
+```
+
+### Step 5b. Run the provided notebook to ingest documents
+
+Start the notebook environment using the below command from within your created `venv` from [Step 1](#step-1-create-a-python-virtual-environment-and-upgrade-pip). Then, open the `doc-ingestion.ipynb` notebook on your web browser.
+
+```bash
+jupyter notebook
+```
+
+Run each cell in order using the "Shift + Enter" shortcut or by clicking the ![play](../assets/jupyter_play_button.png) button.
+
+After the last cell has completed execution, you will see a message like "**Indexed n/n documents**" that indicates that your documents were successfully chunked and ingested into Elasticsearch. 
+If you see an error and only a few of your documents got indexed, you may want to delete the index through the [Kibana dashboard](../ICD_Elasticsearch_install_and_setup.md#step-2-set-up-kibana-to-connect-to-elasticsearch) or using the [delete index API](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-delete-index.html)
+and restart again by attempting with smaller number of documents at a time. 
+
+If you'd like to update the number of documents to send per request in the Elasticsearch API, you can update the 
+`chunk_size` parameter in the call to `helpers.streaming_bulk` method in the final cell of the notebook.
+
+Your documents are now available in the index, ready for searching and querying. Follow the steps outlined below to use this index in a RAG based setup with Watson Assistant. 
+
+**NOTE**: There are some example documents available [here](../assets/sample_pdf_docs), if you would like to test the setup.
+
+### Step 5: Connecting Watson Assistant to Elasticsearch for Conversational Search
+
+There are two options to connect your elasticsearch instance to your assistant for conversational search:
+
+
+
+#### Step 5a. Using the built-in watsonX Assistant Conversational Search
+
+Follow instructions in the [elasticsearch integration documentation](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-elasticsearch-add#setup-elasticsearch) to configure your elasticsearch index for conversational search
+
+**NOTES**
+
+1. In `Step 3` , use `title` for the **Title** field and `text` for the **Text** field
+2. In `Step 4` , configure the custom query body using the snippet below: 
+
+	```
+	{
+	  "query": {
+	    "text_expansion": {
+	      "ml.tokens": {
+	        "model_id": ".elser_model_1",
+	        "model_text": "$QUERY"
+	      }
+	    }
+	  }
+	}
+	```
+
+3. Remember to enable the **conversational search** toggle to `on` to activate it
+
+<img src="../assets/elasticsearch-integration.png" width="400"/>
+
+
+#### Step 5b. Using custom extensions
+
+Follow the steps outlined in guide to [connect your assistant to elasticsearch and watsonx using custom extensions](../../starter-kits/language-model-conversational-search#example-1-connect-your-assistant-to-elasticsearch-and-watsonx-via-custom-extensions) 
```

Comment:
This link is broken.  It needs one more `../` at the start because this README is in `docs/elasticsearch-install-and-setup/python-document-ingestion/` and it needs to get out of all three of those directories.

Reply:
good catch!